### PR TITLE
added requirement for ATP

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#689

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/virus-detection-in-spo.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ description: "Office 365 can help protect your environment from malware by detec
 
 # Virus detection in SharePoint Online
 
+> [!IMPORTANT]
+> To use this feautre, Office 365 Advanced Threat Protection (ATP) is required. To learn more, see [Turn on Office 365 ATP for SharePoint, OneDrive, and Microsoft Teams](https://docs.microsoft.com/microsoft-365/security/office-365-security/turn-on-atp-for-spo-odb-and-teams).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/virus-detection-in-spo.md]:
```suggestion
> To use this feature, Office 365 Advanced Threat Protection (ATP) is required. To learn more, see [Turn on Office 365 ATP for SharePoint, OneDrive, and Microsoft Teams](https://docs.microsoft.com/microsoft-365/security/office-365-security/turn-on-atp-for-spo-odb-and-teams).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - typo

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review based on issue #689.

Thank you.

cc: @kenwith

**Review by MSFTTracyP — APPROVED** [review]
Thanks

**Comment by MaratMussabekov** [comment]
Hello @AntoineJo ,
Thank you very much for the provided information.
Dear @MSFTTracyP , could you please confirm?
Thank you

**Comment by mypil** [comment]
@MSFTTracyP - We would like to follow up on this. Can you please share your insights on this?

Thank you.

**Pull request merged**